index.d.ts

```diff
@@ -27,6 +28,11 @@ declare namespace meow {
 
 	type AnyFlags = {[key: string]: StringFlag | BooleanFlag | NumberFlag};
 
+	/**
+	Callback function to generate a help text you want.
+	*/
+	type GenerateHelp = (args: Readonly<{wholeText: string; flagLines: readonly string[]; description: string; options: Readonly<Options<AnyFlags>>}>) => string;
```

Comment:
The `args` type should be a separate `interface` and each property should have a doc-comment.

Reply:
`args` should have a better name too.